mgmtfn/dockplugin/dockplugin.go

```diff
@@ -39,8 +39,12 @@ import (
 )
 
 const pluginPath = "/run/docker/plugins"
+
+//const pluginPath = "/usr/share/docker/plugins"
 const driverName = "netplugin"
 
+const useUnixSocket = true
```

Comment:
I added it to debug a problem when they changed lib network API from version v1 to v1.1.
Sure. I can remove it.